src/components/CheckboxGroup/CheckboxGroup.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { checkboxGroupDriverFactory } from './CheckboxGroup.driver';
+import { CheckboxGroup } from './';
+import { Checkbox } from '../Checkbox';
+import { checkboxGroupTestkitFactory } from '../../testkit';
+import { checkboxGroupTestkitFactory as enzymeCheckboxGroupTestkitFactory } from '../../testkit/enzyme';
+
+const el = (
+  <CheckboxGroup>
+    <Checkbox
+      name="group1"
+      onChange={val => console.log(val)}
+      label="Checkbox 1️⃣"
+    />
+    <Checkbox
+      name="group1"
+      onChange={val => console.log(val)}
+      label="Checkbox 2️⃣"
+    />
+    <Checkbox
+      name="group1"
+      onChange={val => console.log(val)}
+      label="Checkbox 3️⃣"
+    />
+  </CheckboxGroup>
+);
+
+describe('CheckboxGroup', () => {
+  const createDriver = createUniDriverFactory(checkboxGroupDriverFactory);
+
+  it('should render', async () => {
+    const driver = createDriver(el);
+    expect(await driver.exists()).toBe(true);
+  });
+
```

Comment:
Do you think we should add tests checking that the children are rendered and such?
We can use the Checkbox testkit for that.

Reply:
It's default generated spec :)

Reply:
Yeah, I will add test of children's render

Reply:
Done ✅